app/config/AppConfig.scala

```diff
@@ -120,4 +123,28 @@ class AppConfig @Inject()(override val runModeConfiguration: Configuration, val
   private def feature2Key(feature: Feature): String = s"microservice.services.features.$feature"
 
   private def str2FeatureStatus(str: String): FeatureStatus = FeatureStatus.withName(str)
+
+  private def whitelistConfig(key: String): Seq[String] =
+    Some(
```

Comment:
Just wondering why we have `Some` here?

Reply:
either Some or using Seq with similar situation with splitting the string